src/txdb.h

```diff
@@ -14,7 +14,6 @@
 #include <utility>
 #include <vector>
 
-class CBigNum;
```

Comment:
Any idea why this: https://github.com/bitcoin/bitcoin/commit/df9eb5e14fa8072bc8a82b59e712c2ba36f13f4c#diff-e3b0b9197f4a56f9e534ffafe96fd306L55 
was not pulled with it here? Cause bignum is gonna get moved away so this currently fails building.